### DISCUSSION TITLE
Let me filter by language

### DISCUSSION THREAD

**Discussion by smach** [community/community#14045] [Other Feature Feedback, Questions, & Ideas]
I don’t find it helpful to see repos in languages I don’t use.

**Comment by jfullstackdev** [comment]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)

maybe this will help minimize those, and they are still improving this, so more upcoming updates
